### PR TITLE
allow for global variables

### PR DESCRIPTION
**Pull request by andyburke**
Lets you pass in globals that can be accessed in handlebars templates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,10 @@ Metalsmith(__dirname)
     .use(handlebars({
         pattern: '**/*.hbs', // defaults to '**'
         partials: 'partials', // defaults to 'partials' in the root
-        targetExtension: 'html' // defaults to html
+        targetExtension: 'html' // defaults to html,
```
Comment (mrbalihai) [inline README.md]: This comma needs to be before the comment to be valid if someone copied and pasted.

File: index.js [hunk 2]
```diff
@@ -16,7 +17,7 @@ const main = (options = {}) =>
             .filter(file => match(file, pattern))
             .forEach(file => {
                 const template = Handlebars.compile(files[file].contents.toString()),
-                      data = files[file],
+                      data = Object.assign( {}, globals, files[file] ),
```
Comment (mrbalihai) [inline index.js]: Please could you remove the spacing after opening bracket and before the closing bracket to keep it consistent with the rest of the code?

**Review by mrbalihai — COMMENTED** [review]
Thanks for the PR :-), delighted to accept but please would it be possible for you to do the recommended changes?

**Comment by andyburke** [comment]
Hey, sorry for the delay. I've made the changes as requested.

**Comment by mrbalihai** [comment]
Great, thanks!

**Comment by andyburke** [comment]
Any chance you can push a 1.1.0 version to npm? I'd love to switch back to your npm module vs. my git branch.

**Comment by mrbalihai** [comment]
I've pushed this now, apologies for the delay.

**Comment by andyburke** [comment]
No worries!

**Pull request merged**